src/rdkafka.h

```diff
@@ -7719,11 +7719,12 @@ rd_kafka_ConfigEntry_synonyms(const rd_kafka_ConfigEntry_t *entry,
  * @brief Apache Kafka resource types
  */
 typedef enum rd_kafka_ResourceType_t {
-        RD_KAFKA_RESOURCE_UNKNOWN = 0, /**< Unknown */
-        RD_KAFKA_RESOURCE_ANY     = 1, /**< Any (used for lookups) */
-        RD_KAFKA_RESOURCE_TOPIC   = 2, /**< Topic */
-        RD_KAFKA_RESOURCE_GROUP   = 3, /**< Group */
-        RD_KAFKA_RESOURCE_BROKER  = 4, /**< Broker */
+        RD_KAFKA_RESOURCE_UNKNOWN           = 0, /**< Unknown */
+        RD_KAFKA_RESOURCE_ANY               = 1, /**< Any (used for lookups) */
+        RD_KAFKA_RESOURCE_TOPIC             = 2, /**< Topic */
+        RD_KAFKA_RESOURCE_GROUP             = 3, /**< Group */
+        RD_KAFKA_RESOURCE_BROKER            = 4, /**< Broker */
+        RD_KAFKA_RESOURCE_TRANSACTIONAL_ID  = 5, /**< Transactional ID */
```

Comment:
Add resource name [here](https://github.com/confluentinc/librdkafka/blob/master/src/rdkafka_admin.c#L2827-L2836)

Reply:
I was trying to do that now, but I live in this decade and the earliest version of clang-formatter is 17 :D

Going to have to pull the 10 version from the archives although I have seen https://github.com/confluentinc/librdkafka/pull/4807 which might be a better candidate to merge so that we don't do that dance twice?

Reply:
Well, got the binaries for clang-format in version 10 so, updated it.